### PR TITLE
Feature/gdev90

### PR DESCRIPTION
**Pull request by KerstenBreuer**
Add:
 - a script for generating an openapi.yaml from FastAPI
 - a CI step that checks if openapi.yaml is up to date

Please note, (as #3) this PR depends on the shared lib at https://github.com/ghga-de/ghga-service-chassis-lib/tree/feature/rest which is not yet available at PyPI. Thus to test it now run the following:
```
pip install git+https://github.com/ghga-de/ghga-service-chassis-lib@feature/rest
dev_install
```
This is also why the CI currently fails. Once the shared library is realized, the installation will work as intended.

This PR depends on #1 and #3, please review them first.

### CODE REVIEW

**Review by ac-jorellanaf — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -16,7 +16,8 @@
 FROM python:3.9.6-buster
 
 COPY . /service
-WORKDIR /service
+WORKDIR /servicet
+=======
```
Comment (lkuchenb) [inline Dockerfile:20]: This doesn't look right

File: .github/workflows/static_code_analysis.yaml [hunk 2]
```diff
@@ -38,3 +38,10 @@ jobs:
       - name: Check license header and file
         run: |
           python3 ./.devcontainer/license_checker.py
+      - name: Check if openapi.yaml is up to date
```
Comment (lkuchenb) [inline .github/workflows/static_code_analysis.yaml:41]: So the goal is that we always check in an `openapi.yaml` file that corresponds to the current API? Can we do that with `pre-commit`?

**Review by lkuchenb — APPROVED** [review]
👍 minor comments below

File: .github/workflows/static_code_analysis.yaml (hunk 2 above)
Comment (KerstenBreuer) [reply .github/workflows/static_code_analysis.yaml:41]: Hm, not sure whether this really fits the pre-commit logic. Typically pre-commit is not interpreting connections between multiple file its only running the checks file by file. But I think in principle we can make that happen somehow. If you like you can add a ticket to the backlog.

**Pull request merged**